src/layouts/Home/components/SectionCallout.tsx

```diff
@@ -8,14 +8,13 @@
 import React, { HTMLAttributes } from 'react';
 import styled, { css } from 'styled-components';
 import { getColor } from '@zendeskgarden/react-theming';
-import { XXL, LG } from '@zendeskgarden/react-typography';
+import { XXL, LG, SM } from '@zendeskgarden/react-typography';
 
-export const StyledSectionHeader = styled.div`
+export const StyledSectionHeader = styled(SM)`
```

Comment:
This `xs` styling was intential to align with TOC headers, mobile and desktop nav, and this home page. @talia-eisenberg do we want to change all of this?

Reply:
No, these are specific, intentional, and need to remain the same.